tests/test_core_to_contrib.py

```diff
@@ -184,14 +188,14 @@
         'airflow.providers.apache.druid.hooks.druid.DruidDbApiHook',
         'airflow.hooks.druid_hook.DruidDbApiHook',
     ),
-    (
-        'airflow.providers.apache.hdfs.hooks.hdfs.HDFSHookException',
-        'airflow.hooks.hdfs_hook.HDFSHookException',
-    ),
-    (
-        'airflow.providers.apache.hdfs.hooks.hdfs.HDFSHook',
-        'airflow.hooks.hdfs_hook.HDFSHook',
-    ),
+    # (
```

Comment:
?

Reply:
HDFS tests are failing for me for python 3 because of bad imports. I commented it out for now and I anyhow disabled discovery for HDFS until hdfs installation is fixed. It is broken now.

Reply:
Ok. It was a problem with my environment. I brought back all hdfs tests